Adyen/UI/Form/FormViewController.swift

```diff
@@ -35,7 +36,10 @@ open class FormViewController: UIViewController, AdyenObserver {
     public var requiresKeyboardInput: Bool { formRequiresInputView() }
 
     /// Indicates the `FormViewController` UI styling.
-    public let style: ViewStyle
+    public var style: ViewStyle = FormComponentStyle()
```

Comment:
Can we mark this as `deprecated`? I think this can be applied to old initializers, so we can immediately see the amount of styling changes in the Demo app